ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiActiveContractsService.scala

```diff
@@ -26,33 +28,46 @@ private[apiserver] final class ApiActiveContractsService private (
     implicit executionContext: ExecutionContext,
     protected val mat: Materializer,
     protected val esf: ExecutionSequencerFactory,
-    logCtx: LoggingContext,
+    loggingContext: LoggingContext,
 ) extends ActiveContractsServiceAkkaGrpc
     with GrpcApiService {
 
   private val logger = ContextualizedLogger.get(this.getClass)
 
+  import ApiActiveContractsService.filters
```

Comment:
I'd prefer the import at the top of the scope.

```suggestion
  import ApiActiveContractsService.filters

  private val logger = ContextualizedLogger.get(this.getClass)

```